plugins/inputs/dpdk/dpdk_connector.go

```diff
@@ -92,6 +136,50 @@ func (conn *dpdkConnector) getCommandResponse(fullCommand string) ([]byte, error
 	return buf[:messageLength], nil
 }
 
+// Executes command, parses response and creates/writes metrics from response to accumulator
+func (conn *dpdkConnector) processCommand(log telegraf.Logger, acc telegraf.Accumulator, commandWithParams string, metadataFields []string) {
```

Comment:
We usually do have `acc` as the first parameter. Can you please change the order? Maybe move the `log` to the end?

Reply:
Addressed in: https://github.com/influxdata/telegraf/pull/14308/commits/594fecb38d9f68e33ff03a50172ee132bc5811f6